### PR TITLE
re-organizing sections

### PR DESCRIPTION
**Pull request by edublancas**
## Describe your changes

- re organized docs
- added a new "sharing notebooks" section to showcase our new dashboard sharing feature

## Issue number

Closes #X

## Checklist before requesting a review

- [ ] Performed a self-review of my code
- [ ] Formatted my code with [`pkgmt format`](https://ploomber-contributing.readthedocs.io/en/latest/contributing/pr.html#linting-formatting)
- [ ] Added [tests](https://ploomber-contributing.readthedocs.io/en/latest/contributing/pr.html#testing) (when necessary).
- [ ] Added [docstring](https://ploomber-contributing.readthedocs.io/en/latest/contributing/pr.html#documenting-changes-and-new-features) documentation and update the [changelog](https://ploomber-contributing.readthedocs.io/en/latest/contributing/pr.html#changelog) (when needed)

----
:books: Documentation preview :books:: https://jupysql--722.org.readthedocs.build/en/722/

### CODE REVIEW

**Review by yafimvo — APPROVED** [review]
Looks good and improves readability and navigation!

File: doc/jupyterlab/sharing.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    property=og:locale: en_US
+---
+
+# Sharing notebooks
+
+```{versionadded} 0.1.6
+Notebook sharing requires `jupysql-plugin>=0.1.6`
+```
+
+You can share notebooks with the click of a button using our JupyterLab integration.
```
Comment (neelasha23) [inline doc/jupyterlab/sharing.md:26]: We can add here why and when sharing notebooks will be useful

File: doc/jupyterlab/sharing.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+![](../static/share-notebook.png)
+
+## Installation
+
+```sh
+pip install jupysql-plugin
+```
+
+> **For detailed instructions [click here.](https://docs.cloud.ploomber.io/en/latest/quickstart/signup.html)**
```
Comment (neelasha23) [inline doc/jupyterlab/sharing.md:36]: Should this redirect to signup page or the [JupyterLab plugin](https://docs.cloud.ploomber.io/en/latest/dashboards/jupyterlab-plugin.html) section?

File: doc/jupyterlab/sharing.md (hunk 1 above)
Comment (edublancas) [reply doc/jupyterlab/sharing.md:26]: good call, added

**Review by neelasha23 — APPROVED** [review]

**Pull request merged**